index.html

```diff
@@ -0,0 +1,165 @@
+    </div>
+    
+  </div>
+  <div class="col-11 col-md-2 col-lg-2   text-center  text-light course-div p-3   m-1">
+    <div class="row justify-content-center align-items-center">
+      <img class="border border-light p-2 border-1 col-2 col-md-6 col-lg-6 p-lg-2 p-md-2" src="images/icons/web-development.svg" alt="web-development" height="50px">
+      <h3 class="heading-course m-md-2 mb-md-0 col-4 col-md-12 col-lg-12 " >Development</h3>
+      <p class="col-6 p-0 m-0 col-md-12 col-lg-12 p-md-1">Learn web development online to advance your education and career. Explore these courses.</p>
+    </div>
+     
+  </div>
+  <div class="col-11 col-md-2 col-lg-2   text-center  text-light course-div p-3 p-lg-4 m-1">
+    <div class="row justify-content-center align-items-center">
+      <img class="border border-light p-2 border-1 col-2 col-md-6 col-lg-6 p-lg-2 p-md-2" src="images/icons/IT.svg" alt="Software" height="50px">
+      <h3 class="heading-course m-md-2 mb-md-0 col-4 col-md-12 col-lg-12 " >Software</h3>
+      <p class="col-6 p-0 m-0 col-md-12 col-lg-12 p-md-1">Learn Software Development online at your own pace. Start today with a special offer.</p>
+    </div>
+    
+  </div>
+  <div class="col-11 col-md-2 col-lg-2  text-center text-light course-div p-3   m-1">
+    <div class="row justify-content-center align-items-center">
+      <img class="border border-light p-2 border-1 col-2 col-md-6 col-lg-6 p-lg-2 p-md-2" src="images/icons/Marketing.svg" alt="Marketing" height="50px">
+      <h3 class="heading-course m-md-2 mb-md-0 col-4 col-md-12 col-lg-12 " >Marketing</h3>
+      <p class="col-6 p-0 m-0 col-md-12 col-lg-12 p-md-1">This course will help to get started as a beginner and become a pro at Marketing</p>
+    </div>
+    
+  </div>
+  <div class="col-11 col-md-2 col-lg-2  text-center text-light course-div p-3   m-1">
+    <div class="row justify-content-center align-items-center">
+      <img class="border border-light p-2 border-1 col-2 col-md-6 col-lg-6 p-lg-2 p-md-2" src="images/icons/business.svg" alt="Business" height="50px">
+      <h3 class="heading-course m-md-2 mb-md-0 col-4 col-md-12 col-lg-12 " >Business</h3>
+      <p class="col-6 p-0 m-0 col-md-12 col-lg-12 p-md-1">On this course, you'll develop a range of business management skills in the world of work.</p>
+    </div>
+    
+  </div>
+ </div>
+ <div class="row justify-content-center">
+ <div class="col-md-6 col-lg-6 col-10 text-center p-5 ">
+ <a href="#" class="text-light d-none d-md-block d-lg-block">SEE THE WHOLE COURSE</a>
+ <button class="d-block d-md-none d-lg-none mx-auto btn apply text-light p-3">Apply Now in techrotics</button>
+ </div>
+ </div>
+</section> 
+<section class="Featured-courses ">
+  <div class="row justify-content-center ">
+    <div class="col-11 d-flex flex-column align-items-center mt-5">
+      <h2 class="text-center text-dark ">Featured courses</h2>
```

Comment:
- [ ] Looking at the mobile version of the `Featured section` from the given design, There is a **`more button`** below the speakers and only two speakers are displayed, as you are meant to replicate the given design, I'll suggest you implement this in your own project as its part of the requirement. :+1: :100: :smile: 
- - Screenshot for reference:
![Screenshot from 2023-05-23 15-05-03](https://github.com/batoolfatima2135/Capstone/assets/107632780/b0ec9989-9392-4351-a908-bdb7cd00ac4b)